### PR TITLE
Feat/api axios types

### PR DESCRIPTION
**Pull request by jordan-a-young**
This PR contains a declaration file for `@availity/api-axios`. I don't think it is 100% perfect, but my hope is this can serve as a starting point and the types can be refined as we go. The resources do not have their specific response shape added, but the user can add it on their end to get around this for now.

I added formatting scripts for `@availity/api-axios` and `@availity/api-core` and ran them. 

Finally, I tried to cleanup some of the docs. This mostly involved formatting and how we display responses. eg: switch from "\```js" to "\```json"

### CODE REVIEW

**Review by danielpickett — APPROVED** [review]

File: docusaurus/docs/resources/native-form.md [hunk 1]
```diff
@@ -29,23 +29,23 @@ nativeForm(spaceId[, params[, formAttributes][, type]]);
...
+- spaceId: String
 
 ### Optional params
 
--   params: Object. Additional parameters you want sent in the post.
--   formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
--   type: String. Override the magneto integration type. Defaults to "saml"
+- params: Object. Additional parameters you want sent in the post.
+- formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
+- type: String. Override the magneto integration type. Defaults to "saml"
```
Comment (chrishavekost) [inline docusaurus/docs/resources/native-form.md:38]: As of 4.0.0 this no longer defaults to `saml`, I missed these docs in my PR.

File: docusaurus/docs/resources/native-form.md [hunk 2]
```diff
@@ -29,23 +29,23 @@ nativeForm(spaceId[, params[, formAttributes][, type]]);
...
--   spaceId: String
+- spaceId: String
 
 ### Optional params
 
--   params: Object. Additional parameters you want sent in the post.
--   formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
--   type: String. Override the magneto integration type. Defaults to "saml"
+- params: Object. Additional parameters you want sent in the post.
+- formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
```
Comment (chrishavekost) [inline docusaurus/docs/resources/native-form.md:37]: `action` now defaults to `/ms/api/availity/internal/spc/magneto/sso/v1/${typeLower}/${spaceId}`
where `typeLower` is either `saml` or `openid`

File: docusaurus/docs/resources/native-form.md (hunk 1 above)
Comment (chrishavekost) [reply docusaurus/docs/resources/native-form.md:38]: There's also one more param now for `clientId` which is optional and defaults to `'clientId'` if not provided.

File: docusaurus/docs/resources/native-form.md [hunk 3]
```diff
@@ -29,23 +29,23 @@ nativeForm(spaceId[, params[, formAttributes][, type]]);
...
+- type: String. Override the magneto integration type. Defaults to "saml"
 
 ###
 
 ```js
 import nativeForm from '@availity/native-form';
 
 nativeForm(
```
Comment (chrishavekost) [inline docusaurus/docs/resources/native-form.md:45]: `nativeForm` is now async so this should be `await nativeForm`
There's also the possibility of errors/promise rejections now that we are (potentially) fetching the proper sso type, so we should probably document this inside a try/catch with error handling.

**Review by chrishavekost — CHANGES_REQUESTED** [review]

**Review by chrishavekost — APPROVED** [review]
🏆

**Pull request merged**